README.md

```diff
@@ -66,108 +78,5 @@ export ARTIFACTORY_PASSWORD=Long_string_of_gibberish_abcdefghijklmnopqrstuvwxyzA
 - `ARTIFACTORY_PASSWORD`:
   This is your **API Key**, *NOT* your password. Find your API key at the bottom of **Authentication Settings** in your profile.
 
```

Comment:
I could use an explanation of how things will work differently now. I thought we would still be doing live-preview on the docs.daml.com repo? 

And wouldn't we still need to reindex the site search?

Reply:
The rest of the file has been moved to `docs/2.6.0/README.md`, so it's not really removed. As stated above, specific instructions for how to build a specific version of the docs may vary per version, so they're in the version-specific README.

You're right that the reindex instructions should stay in the top-level README, though.

Reply:
>  I thought we would still be doing live-preview on the docs.daml.com repo?

Sorry I forgot to answer this part specifically. You can still run `live-preview`, but you have to run it for a specific version, and the way in which you run it may depend on the version. Specifically, right now (but this may change over time):

- If you want a live preview of the 2.6.0 version, you navigate to `docs/2.6.0` then run the `live-preview` command.
- If you want a live preview of the 2.5.1 or 2.5.2 versions, you navigate to their respective folders (`docs/2.5.1` or `docs/2.5.2`) and run `docs/live-preview.sh`.